src/lib/sfmc_plugin_platform_interface.dart

```diff
@@ -33,6 +34,9 @@ abstract class SfmcPluginPlatform extends PlatformInterface {
         delayRegistration: delayRegistration);
   }
 
+  setHandler(Future<dynamic> Function(MethodCall call)? handler) =>
```

Comment:
please update the example as well. thanks

Reply:
Updated